I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 348, in script
    jump scene_5_trip_to_destination
ScriptError: could not find label 'scene_5_trip_to_destination'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 348, in script
    jump scene_5_trip_to_destination
  File "C:\Users\User\Desktop\renpy-8.2.3-sdk\renpy\ast.py", line 1701, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\User\Desktop\renpy-8.2.3-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'scene_5_trip_to_destination'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.3.24061702
Brainrot Visual Novel Name Pending 1.0
Wed Oct 16 08:40:25 2024
